src/components/Dialog/Dialog.js

```diff
@@ -126,32 +116,21 @@ Dialog.propTypes = {
    * Determine the container for scrolling the dialog.
    */
   scroll: PropTypes.oneOf(['body', 'paper']),
-  /**
- * Enter event to use with UXPin interactions.
- */
-  onEnter: PropTypes.func,
 
-  /**
-   * Exit event to use with UXPin interactions.
-   */
-  onExit: PropTypes.func,
-  
   onClose: PropTypes.func,
 
   /**
- * If `true`, the dialog stretches to `maxWidth`.
- * @uxpinignoreprop
- */
+   * If `true`, the dialog stretches to `maxWidth`.
+   * @uxpinignoreprop
+   */
   fullWidth: PropTypes.bool,
```

Comment:
If we use the use `uxpinuseportal` then I think we can remove this as it will always use the full screen.

If we keep this prop then it the top Preview bar hides some of the component.

Reply:
I think stretch to maxWidth means the modal itself takes all the available space, while the "use portal" feature is about the black overlat that takes all the screen, it's two different things. 

So maybe we could try to enable this prop instead of ignoring it like we do know, what do you think?

Reply:
Maybe there was a confusion between `fullScreen` (I think that was the property you checked) and `fullWidth` (that was not available because of `uxpinignorerprop`).

I can see the problem of layout when `fullWidth` is enabled, I'm going to fix it by using `position: absolute` instead of `position: fixed`.

Reply:
Yea, that fixes it 😄. Good catch! 

I'm interested in why removing the previous style css and simple adding changing fixed to absolute fixes the issue of hiding part of the component behind the topbar.

Reply:
In theory we don't need inline styles to change the styling of MUI components we should use either the component properties (`maxWidth` when they are available) or the `sx` property.